### DISCUSSION TITLE
Meeting List with Divi and Shortcode?

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1352] [Q&A] [answered]
We are using the Meeting List plugin with Divi theme and page builder.

I wanted to add a graphic page title to the Meetings page. I attempted this by building a template with the Divi Theme Builder and the shortcode [tsml_ui] and this is what I got.
<img width="1212" alt="Monosnap CECDA Staging 2024-02-14 21-10-26" src="https://github.com/code4recovery/12-step-meeting-list/assets/159723102/33caf6f8-34ae-4492-a040-c0250994e670">

I was surprised to see that most of the filters from the default view have disappeared, we seem to be left with just the "Type" filter, not location, day or time.  The region for each meeting and the map view are also missing.

Is there a way to configure the displayed filters for the shortcode? Is there another shortcode?

I have seen some other sites that have added a page title above the archive without changing the display or filters of the meeting list.  Is there another way to do this in Divi that I am not thinking of?

**Bonus question**: is there a way to suppress the display of the title "Meetings" which is currently part of the meeting list itself? If we add the graphic header, it is no longer needed.

Thank you all so much!

**Comment by joshreisner** [comment]
hi, it from the image like you have only entered two meetings, both on the same day and time, which is why those menus aren't showing yet. have you added any regions yet? you'll need a few of those for that menu to show up as well

and the maps link i think won't show until you enter a mapbox key on the meetings > settings page

you should be able to hide the title with 

```js
<script>
  var tsml_react_config = {
    show: { title: false },
  };
</script>
```

**Reply by willinrecovery** [reply]
Thanks Josh, this looks super helpful.  

Where do I actually put that script?

**Reply by kiyote33** [reply]
I think either 
.tsml-ui h1 { display:none; } 
or 
.tsml-ui h1 { visibility:hidden; } 

would work in the Additional CSS if your divi theme has that feature.

**Reply by joshreisner** [reply]
yeah sorry on second thought that code i suggested won't work when using the shortcode. i think we also might be taking that option out in a future release. i think you'd be better off with the CSS approach…

**Reply by willinrecovery** [reply, marked as answer]
Thank you @joshreisner  and @kiyote33.

Based on Josh's most recent comment, I gave up on trying not to generate the h1 at all. To avoid having two h1s, I changed the visible "h1" to an h2 and made the actual h1 generated by the plugin invisible.

I had added the meeting list to the page using the shortcode in a Divi code module.

I then added the following in the custom CSS > Free-Form CSS panel: selector h1 { display: none; }

<img width="610" alt="Monosnap CECDA Staging 0 2024-02-17 15-45-39" src="https://github.com/code4recovery/12-step-meeting-list/assets/159723102/5e4c27b5-248d-4764-bf40-9784dcd3954a">

This successfully made the h1 invisible and now the visible header looks like an h1 but is really an h2.

<img width="1224" alt="Monosnap CECDA Staging 0 2024-02-17 15-50-17" src="https://github.com/code4recovery/12-step-meeting-list/assets/159723102/4f380b57-4933-4e88-b77a-168a6f614dea">

Thanks so much to everybody for your service. This plugin is such a great help!

**Comment by ginslo** [comment]
Hello. I'll chime in briefly. Hiding the h1 "Meetings" title *may* be possible but as I understand it, it's outside the normal use case of TSML. I've had success when applying this css code in devtools:

```
#tsml-ui h1[aria-live="polite"] {
  display: none;
}
```

But have not been able to make it work on a permanent basis when adding the same code to the website css files, at least with my theme (EmpowerWP). Your mileage may differ.

**Reply by willinrecovery** [reply]
Thank you for the tip, Ginslo. I will try that too.

Would you happen to know where that script that Josh suggested would go?

I realized that if I leave both "Meetings" headlines (even if one is invisible) there will be two h1's.

Thank you so much!

**Reply by willinrecovery** [reply]
Thank you again, @ginslo 

I used a modified version of your suggestion as I am using Divi which has particular features and syntax if you add the CSS within a specific module.

I gave details above in case you are interested.

**Reply by ginslo** [reply]
Happy to help and am glad you got it to work!
